tests/api/test_view.py

```diff
@@ -52,16 +52,14 @@ def test_batch_queries(category, product, api_client):
     assert data["category"]["name"] == category.name
 
 
-def test_graphql_view_get_in_non_debug_mode(client):
+def test_graphql_view_get_enabled_or_disabled(client, settings):
```

Comment:
I usually prefer to keep each case as separate test so in case of failure I don't have to analyze the logic to see whats wrong. But since it's so trivial can be left as is I guess.